xds/src/main/java/io/grpc/xds/XdsChannelCredentials.java

```diff
@@ -24,7 +24,7 @@
 import io.grpc.netty.InternalProtocolNegotiator;
 import io.grpc.xds.internal.sds.SdsProtocolNegotiators;
 
-@ExperimentalApi("https://github.com/grpc/grpc-java/issues/7479")
+@ExperimentalApi("https://github.com/grpc/grpc-java/issues/7514")
```

Comment:
https://github.com/grpc/grpc-java/issues/7514 is really for XdsServerBuilder which can also be applied to `XdsServerCredentials`. Should `XdsChannelCredentials` also be included under that issue? We can modify the issue to also include all xDS functionality in the API.

Reply:
Yeah, it is a bit weird, but it is all the "xds security stuff" which will be stabilized at the same time. I agree XdsServerCredentials is a cleaner fit, but I was thinking "it doesn't matter much, just throw them together as it makes things easier for everyone."

Reply:
I am okay with using #7514 for this. But it will be good to update the description to properly include XdsChannelCredentials functionality.

Reply:
I've updated the #7514.